bindings/c/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "c"
+version = "0.3.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+name = "safetensors"
+crate-type = ["staticlib"]
```

Comment:
Any specific reason to build static libs instead of a shared one ? (Ideally I imagine user could switch easily if they wanted). I think shared is a saner default probably, no ?